frappe/website/doctype/web_template/web_template.js

```diff
@@ -2,11 +2,21 @@
 // For license information, please see license.txt
 
 frappe.ui.form.on('Web Template', {
-	refresh(frm) {
+	refresh: function(frm) {
 		if (!frappe.boot.developer_mode && frm.doc.standard) {
 			frm.disable_form();
 		}
 
 		frm.toggle_display('standard', frappe.boot.developer_mode);
+		frm.toggle_display('template', !frm.doc.standard);
+	},
+	standard: function(frm) {
+		if (!frm.doc.standard) {
+			// If standard changes from true to false, hide template until
+			// the next save. Changes will get overwritten from the backend
+			// on save and should not be possible in the UI.
+			frm.toggle_display('template', false);
+			frm.save();
```

Comment:
Why trigger Save automatically? Let the user save the form?

Reply:
We could show an alert "Please save to load the template." instead. What do you think makes for a better UX?

Reply:
@netchampfaris which one do you prefer?

Reply:
Personally, I don't think we need to do anything. But, if you want to add a message, add it using `frm.dashboard.set_headline`